brownie/allocations.py

```diff
@@ -199,17 +199,3 @@ def transactions_for_reallocation(votes):
     txs = wrap_in_loss_prevention(txs, vault_loss_raw)
 
     return txs
-
-
```

Comment:
No need for an example at the bottom of the file, now that we have the run logs.